h/zis/services/auth.h

```diff
@@ -43,6 +43,23 @@ typedef struct AbendInfo_tag {
 #define ZIS_AUTH_SERVICE_PASSWORD_MAX_LENGTH    100
 #define ZIS_AUTH_SERVICE_CLASS_MAX_LENGTH       8
 #define ZIS_AUTH_SERVICE_ENTITY_MAX_LENGTH      255
+#define ZIS_AUTH_SERVICE_APPL_MAX_LENGTH     8
+
+#define ZIS_SAF_IDT_SERVICE_CURRENT_VERSION         1
+
+/* The #define for ZIS_AUTH_SERVICE_SAFIDT_OPTION_GENERATE_IDT has been deprecated, and will be removed
```

Comment:
Does it make sense to have this comment in `AuthServiceParmList` when `options` and `ZIS_AUTH_SERVICE_SAFIDT_OPTION_GENERATE_IDT` used to be?